programs/account-compression/src/instructions/nullify_leaves.rs

```diff
@@ -63,18 +63,13 @@ pub fn process_nullify_leaves<'a, 'b, 'c: 'info, 'info>(
         msg!("only implemented for 1 nullifier update");
         return Err(AccountCompressionErrorCode::NumberOfProofsMismatch.into());
     }
-    let changelog_event = insert_nullifier(
+    insert_nullifier(
         proofs,
         change_log_indices,
         leaves_queue_indices,
         indices,
         ctx,
     )?;
-    emit_indexer_event(
-        changelog_event.try_to_vec()?,
-        &ctx.accounts.log_wrapper,
-        &ctx.accounts.authority,
-    )?;
```

Comment:
why move this into insert_nullifier?

Reply:
To avoid returning the vector. If we keep it in one function, it's going to be dropped once the function exits

Reply:
It's the end of the transaction so it shouldn't change anything right?